action.yml

```diff
@@ -65,17 +60,22 @@ inputs:
       in which case LuaRocks will fall back to this one.
     required: false
 runs:
-  using: "docker"
-  image: "Dockerfile"
-  args:
-    - ${{ inputs.name }}
-    - ${{ inputs.version }}
-    - ${{ inputs.dependencies }}
-    - ${{ inputs.labels }}
-    - ${{ inputs.copy-directories }}
-    - ${{ inputs.summary }}
-    - ${{ inputs.detailed_description }}
-    - ${{ inputs.build_type }}
-    - ${{ inputs.template }}
-    - ${{ inputs.upload }}
-    - ${{ inputs.license }}
+  using: "composite"
+  steps:
+    - uses: cachix/install-nix-action@v20
+    - run: nix profile install "${{ github.action_path }}#luarocks-tag-release-action"
+      shell: bash
+    - run: luarocks-tag-release-action
+      env:
```

Comment:
why have you moved them to the env ?

Reply:
It's a workaround for https://github.com/actions/runner/issues/665